src/lib/widgets/bottom-bar/ui/bottom-bar.svelte

```diff
@@ -29,7 +29,7 @@
 >
   <BottomBarNavItem name="Gallery" icon={IconVideoLibrary} href="/" showLabel={!hasAudio} />
   {#if hasAudio}
-    <MiniPlayer on:click={() => dispatch('miniplayer-click')} />
+    <MiniPlayer on:expand={() => dispatch('miniplayer-click')} />
```

Comment:
Why did you rename one event but not the other? (referring to miniplayer-**click** event)

Reply:
Couldn't be bothered refactoring more than what was necessary. Also to keep the diff clean. I don't think this is a problem